development/verify-locale-strings.js

```diff
@@ -10,87 +10,88 @@
 //
 ////////////////////////////////////////////////////////////////////////////////
 
-var fs = require('fs')
-var path = require('path')
+const fs = require('fs')
+const path = require('path')
+const localeIndex = require('../app/_locales/index.json')
```

Comment:
If we are requiring this index file to be used, we should [update the doc file too](https://github.com/MetaMask/metamask-extension/blob/master/docs/translating-guide.md).